### DISCUSSION TITLE
Starting today we have Stefanini scraper

### DISCUSSION THREAD

**Discussion by irina-adam** [peviitor-ro/community#5] [FrontEnd Team]
Scraperul se ruleaza de pe repo-ul Irinei Adam:
https://github.com/irina-adam/stefanini
Cheers!

**Comment by sebiboga** [comment]
ai setat sa ruleze automat?
la ce ora porneste? @irina-adam

**Reply by irina-adam** [reply]
Dada, ora 20:30 in Romania
